### PR TITLE
[ADD] account_invoice_refund_line

### PR DESCRIPTION
**Pull request by jarroyomorales**
This module allows the user no refund specific lines in a invoice

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Better call it `account_invoice_refund_line_selection`. I still need to check how the selection is done for being a real UX improvement over generating the draft invoice and remove or change there the created lines.

**Comment by JordiBForgeFlow** [comment]
You can see the real improvement when you have large invoices, and youbwant
just to refund 1 line.

File: account_invoice_refund_line/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{
+    'name': 'Account invoice refund line',
+    'version': '11.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'summary': 'This module allows the user no refund specific '
+               'lines in a invoice',
+    'author': 'Creu Blanca,'
+              'Odoo Community Association (OCA)',
+    'maintainer': 'OCA',
+    'website': '',
```
Comment (JordiBForgeFlow) [inline account_invoice_refund_line/__manifest__.py]: add `https://github.com/OCA/account-invoicing`

File: account_invoice_refund_line/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Creu Blanca
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': 'Account invoice refund line',
+    'version': '11.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'summary': 'This module allows the user no refund specific '
```
Comment (JordiBForgeFlow) [inline account_invoice_refund_line/__manifest__.py]: no refund --> to refund

File: account_invoice_refund_line/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+               'lines in a invoice',
+    'author': 'Creu Blanca,'
+              'Odoo Community Association (OCA)',
+    'maintainer': 'OCA',
```
Comment (JordiBForgeFlow) [inline account_invoice_refund_line/__manifest__.py]: remove this line if you don't want to put your github user as maintainer.

File: account_invoice_refund_line/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    'website': '',
+    'license': 'AGPL-3',
+    'depends': ['account'],
+    'data': ['wizards/account_invoice_refund_view.xml'],
+    'demo': [],
```
Comment (JordiBForgeFlow) [inline account_invoice_refund_line/__manifest__.py]: Remove if empty

File: account_invoice_refund_line/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    'name': 'Account invoice refund line',
+    'version': '11.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'summary': 'This module allows the user no refund specific '
+               'lines in a invoice',
+    'author': 'Creu Blanca,'
+              'Odoo Community Association (OCA)',
+    'maintainer': 'OCA',
+    'website': '',
+    'license': 'AGPL-3',
```
Comment (JordiBForgeFlow) [inline account_invoice_refund_line/__manifest__.py]: LGPL, consistent with the other headers.

File: account_invoice_refund_line/models/account_invoice.py [hunk 6]
```diff
@@ -0,0 +1,78 @@
+# Copyright 2019 Creu Blanca
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models, _
+
+# mapping invoice type to refund type
+TYPE2REFUND = {
```
Comment (JordiBForgeFlow) [inline account_invoice_refund_line/models/account_invoice.py]: istead of this, import it as:
`from odoo.addons.account.models.account_invoice import TYPE2REFUND`

**Review by JordiBForgeFlow — CHANGES_REQUESTED** [review]
minor changes.

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by bjeficent — APPROVED** [review]
Functional testing, LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tested on runbot. LGTM.

**Pull request merged**